src/features/BlankScreen011967/screens/blank.js

```diff
@@ -10,54 +10,65 @@ import {
   Button,
   Switch,
   TextInput,
-  StyleSheet
-} from "react-native";
+  StyleSheet,
+} from 'react-native';
 class Blank extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      email: "",
-      password: ""
+      email: '',
+      password: '',
     };
   }
   render = () => (
     <View style={styles.container}>
       <Text>Welcome to Crowdbotics</Text>
       <TextInput
-        onChangeText={email => this.setState({ email: email })}
-        style={{ height: 40, borderColor: "gray", borderWidth: 1 }}
+        onChangeText={email => this.setState({email: email})}
+        style={{height: 40, borderColor: 'gray', borderWidth: 1}}
       />
       <TextInput
-        onChangeText={password => this.setState({ password: password })}
-        style={{ height: 40, borderColor: "gray", borderWidth: 1 }}
+        onChangeText={password => this.setState({password: password})}
+        style={{height: 40, borderColor: 'gray', borderWidth: 1}}
       />
       <TouchableOpacity
         style={styles.button}
-        onPress={() => this.props.signup(this.state.email, this.state.password)}
-      >
+        onPress={() =>
+          this.props.signup(this.state.email, this.state.password)
+        }>
         <Text>Signup</Text>
       </TouchableOpacity>
+      <Text>User count {this.props.users.length}</Text>
+      <Text>Errors {this.props.registration.message}</Text>
     </View>
   );
 }
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    marginHorizontal: 16
+    marginHorizontal: 16,
   },
   button: {
-    alignItems: "center",
-    backgroundColor: "#DDDDDD",
-    padding: 10
-  }
+    alignItems: 'center',
+    backgroundColor: '#DDDDDD',
+    padding: 10,
+  },
 });
 function mapStateToProps(state) {
-  return { state: state };
+  return {
+    users: state.apiReducer.users,
+    registration: state.apiReducer.registration,
```

Comment:
This gives access to part of the state tree to this screen. Notice the reducer consequents return new objects for those keys (through `Object.assign`)